### PR TITLE
1981 lifecycle diagram

### PR DESCRIPTION
**Pull request by jordanpowell88**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/store/index.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ Store is RxJS powered state management for Angular applications, inspired by Red
 
 Detailed installation instructions can be found on the [Installation](guide/store/install) page.
 
+## Diagram
+<figure>
+  <img src="../../images/guide/data/state-management-lifecycle.png" alt="NGRX State Management Lifecycle Diagram">
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/store/index.md]:
```suggestion
  <img src="generated/images/guide/store/state-management-lifecycle.png" alt="NGRX State Management Lifecycle Diagram">
```

**Comment by brandonroberts** [comment]
Thanks @jordanpowell88! We need a bit of context on the page with the image explaining what it is. The image also needs to be set to 100% width so it fits on the page correctly.

File: projects/ngrx.io/content/guide/store/index.md [hunk 2]
```diff
@@ -13,6 +13,13 @@ Store is RxJS powered state management for Angular applications, inspired by Red
 
 Detailed installation instructions can be found on the [Installation](guide/store/install) page.
 
+## Diagram
+
+The following diagram represents the overall general flow of application state in ngrx.
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/store/index.md]: Nits: use NgRx when referring to the platform

Comment (jordanpowell88) [reply projects/ngrx.io/content/guide/store/index.md]: Just pushed fix

**Pull request merged**